src/test/java/ru/training/at/hw2/ex1/FirstExerciseTest.java

```diff
@@ -0,0 +1,79 @@
+package ru.training.at.hw2.ex1;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import ru.training.at.hw2.TestsInitialization;
+
+public class FirstExerciseTest extends TestsInitialization {
```

Comment:
You have one point in HW description about class and method naming. It should be named accordingly to functionality.